style.css

```diff
@@ -0,0 +1,118 @@
+section {
+  height: 718px;
+  width: 375px;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: 0;
+}
+
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.toolbar .logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: 1 1 40%;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 20px;
+  padding-top: 10px;
+}
+
+.toolbar img {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-right: 20px;
+  padding-top: 10px;
+}
```

Comment:
[OPTIONAL]

Please don't copy paste the CSS from the Figma design, yes you should follow it but it's better to write it on your own so you understand what each line does, for example in this block of code, there are some lines that aren't necessary, like `flex: none;` I suggest you go to your developer tools in the browser check each element you have and try to disable some of the lines of CSS to see what is essential and what each line does, this should help you learn more about this! 😁